### PR TITLE
Sync button

### PR DESCRIPTION
**Pull request by leplatrem**
- [x] Demo synchronisation usage with a dummy sync button
- [x] Gray list and button during sync

@n1k0 r?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,6 +2,11 @@
 <html>
 <head>
   <title>Kinto React Boilerplate</title>
+  <style>
+    .disabled {
+      color: GrayText;
+    }
+  </style>
```
Comment (n1k0) [inline index.html]: How about moving this to some external stylesheet? It's a boilerplate after all.

File: scripts/components/App.js [hunk 2]
```diff
@@ -60,8 +60,15 @@ export class Store extends EventEmitter {
       .then(res => {
         this.state.items.push(res.data);
         this.emit('change', this.state);
-      })
```
Comment (n1k0) [inline scripts/components/App.js]: It's probably good to keep catching errors. Maybe we could trigger an error event with the details, so subscribed components could render a fancy flash message or something?

File: test/index_test.js [hunk 3]
```diff
@@ -56,23 +72,40 @@ describe("App", () => {
 
   describe("Store", () => {
 
+    beforeEach((done) => {
+      sandbox.stub(store.collection, 'create')
+        .returns(Promise.resolve({data: {label: "Hola!"}}));
```
Comment (n1k0) [inline test/index_test.js]: You should add dedicated `describe` blocks for each public methods, and move specific stubbing down there.

File: scripts/components/App.js [hunk 4]
```diff
@@ -60,8 +60,15 @@ export class Store extends EventEmitter {
...
-      .catch(console.error.bind(console));
+      });
+  }
+
+  sync() {
+    return this.collection.sync()
+      .then(res => {
+        this.state.items = this.state.items.concat(res.created);
+        this.emit('change', this.state);
```
Comment (n1k0) [inline scripts/components/App.js]: Nit: I think we went with double quotes for strings. If you prefer simple ones, that's fine; just ensure all double ones are updated.

**Comment by n1k0** [comment]
Looks good. r=wc

File: index.html (hunk 1 above)
Comment (almet) [inline index.html]: +1

**Pull request merged**